docs/pages/stacks.md

```diff
@@ -0,0 +1,84 @@
+- [The Indie Stack](https://github.com/remix-run/indie-stack): Deployed to a long-running Node.js server with a persistent SQLite database. Intended to get you going quickly with little complexity for prototypes and proof of concepts that can be later updated to the Blues stack without much trouble.
+- [The Grunge Stack](https://github.com/remix-run/grunge-stack): Deployed to a serverless function running Node.js with DynamoDB for persistance. Intended for folks who want to deploy a production-grade application on AWS infrastructure serving millions of users.
+
+Yes, these are named after music genres. 🤘 Rock on.
+
+There will be more stacks available in the future. And you can make your own (and we strongly encourage it)!
+
+## Custom Stacks
+
+The Remix CLI will help you get started with one of these built-in stacks, but if you want, you can create your own stack and the Remix CLI will help you get started with that stack. There are several ways to do this, but the most straightforward is to create a GitHub repo:
+
+```
+npx create-remix@latest --template my-username/my-repo
+```
+
+Custom stacks give an enormous amount of power and flexibility and we hope you create your own that suites the preferences of you and your organization (feel free to fork ours!).
+
+### `--template`
+
+The template option can be any of the following values:
+
+- The name of a stack in the remix-run GH org (e.g. `blues-stack`)
+- A GH username/repo combo (e.g. `mcansh/snkrs`)
+- A file path to a directory on disk (e.g. `/my/remix-stack`)
+- A path to a tarball on disk (e.g. `/my/remix-stack.tar.gz`)
+- A URL to a tarball (e.g. `https://example.com/remix-stack.tar.gz`)
+- A file URL (e.g. `file:///Users/michael/remix-stack.tar.gz`)
+
+Additionally, if your stack is in a private GitHub repo, you can set a GitHub token in the `GITHUB_TOKEN` environment variable:
+
+```
+GITHUB_TOKEN=yourtoken npx create-remix@latest --template your-private/repo
+```
+
+The [token just needs `repo` access][repo access token].
+
+### Custom Template Tips
+
+#### Remix deps
+
+If you set the any dependencies in package.json to `*`, the Remix CLI will change it to a semver carrot of the latest released version:
+
+```diff
+-   "remix": "*",
++   "remix": "^1.2.3",
+```
+
+This allows you to not have to regularly update your template to the latest version of that specific package. Of course you do not have to put `*` if you'd prefer to manually manage the version for that package.
+
+#### `remix.init/index.js`
+
+If the template has a `remix.init.js` file at the root then that file will be executed after the project has been generated and dependencies have been installed. This gives you a chance to do anything you'd like as part of the initialization of your template. For example, in the blues stack, the `app` property has to be globally unique so we use the `remix.init/index.js` file to change it to the name of the directory that was created for the project + a couple random characters.
```

Comment:
We aren't supporting `remix.init.js`. Just `remix.init/index.js`. This will keep init script dependencies out of the main app's `package.json`.

Reply:
Will fix 👍